internal/acceptance/data_metastores_test.go

```diff
@@ -0,0 +1,16 @@
+package acceptance
+
+import (
+	"testing"
+)
+
+func TestUcAccDataSourceMetastore(t *testing.T) {
+	accountLevel(t, step{
+		Template: `
+		data "databricks_metastores" "this" {}
```

Comment:
This is a reasonable integration test for the "no metastores" case. Can we add an integration test for the "one or more metastores case"?

Reply:
@mgyucht not sure how easy it is to test both scenarios in a single account, unless we add a filter option to this data source

Reply:
Check has been added in the test